src/site/content/en/blog/min-max-clamp/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'min(), max(), and clamp(): Three Logical CSS Functions to Use Today'
+subhead: Learn how to control element sizing, maintain proper spacing, and implement fluid typography using these well-supported CSS functions.
+authors:
+  - una
+date: 2020-10-14
+hero: hero.jpg
+alt: Set of tools on a desk. Via @yer_a_wizard on Unsplash.
+description: Min, max, and clamp provide some powerful CSS capabilities that enable more responsive styling with fewer liens of code. This post goes over how to control element sizing, maintain proper spacing, and implement fluid typography using these well-supported CSS math functions.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - css
+  - chrome79
+feedback:
+  - api
+---
+
+With responsive design evolving and becoming increasingly nuanced, CSS itself is constantly evolving and providing authors increased control. The [`min()`](https://developer.mozilla.org/en-US/docs/Web/CSS/min), [`max()`](https://developer.mozilla.org/en-US/docs/Web/CSS/max), and [`clamp()`](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp) functions, now supported in all modern browsers, are among the latest tools in making authoring websites and apps more dynamic and responsive. 
+
+When it comes to flexible and fluid typography, controlled element resizing, and maintaining proper spacing, `min()`, `max()`, and `clamp()` can help.
+
+## Background
+
+The math functions, `calc()`, `min()`, `max()`, and `clamp()` allow mathematical expressions with addition (+), subtraction (-), multiplication (*), and division (/) to be used as component values — [[CSS Values And Units Level 4]](https://www.w3.org/TR/css-values-4/#calc-notation). 
+
+Safari was the first to [ship](https://bugs.webkit.org/show_bug.cgi?id=167000) the complete set of functions in April 2019, with Chromium following later that year in version 79. This year, with Firefox [75](https://bugzilla.mozilla.org/show_bug.cgi?id=1519519) shipping, we now have browser parity for `min()`, `max()`, and `clamp()` in all evergreen browsers.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./caniuse.png" alt="" />
```

Comment:
Empty alt text.

Reply:
I left this empty because the `figcaption` describes the image